### ISSUE TITLE
Secondary keys not unique in tf-2.hmm

### ISSUE THREAD

**Issue by mpunzalan** [linnabrown/run_dbcan#3]
Hi!

I hope you can help me with this issue:

hmmpress tf-2.hmm 
Working...    SSI index construction failed:
 secondary keys not unique: '103612' occurs more than once

Thanks!

**Comment by linnabrown** [comment]
>This problem could be solved by downgrading your HMMER to 3.1b2. I think new HMMER (3.2) wants more stringency and the tf-2.hmm needs updating to work with the new HMMER.

**Comment by linnabrown** [comment]
I am now updating tf-2.hmm, one moment please

**Comment by mpunzalan** [comment]
Thank you very much! I'll wait for the update

**Comment by linnabrown** [comment]
Done.

**Comment by mpunzalan** [comment]
After the update, I tried the hmmpress again using hmmer 3.2.1 and I encountered the same problem.

I removed the 3.2.1 package and downgraded into hmmer 3.1b2 and the hmmpress of tf-2.hmm worked!

hmmpress tf-2.hmm 
Working...    done.
Pressed and indexed 165 HMMs (165 names and 165 accessions).
Models pressed into binary file:   tf-2.hmm.h3m
SSI index for binary model file:   tf-2.hmm.h3i
Profiles (MSV part) pressed into:  tf-2.hmm.h3f
Profiles (remainder) pressed into: tf-2.hmm.h3p

Thank you very much!
